CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+
+## [0.6.0]
+
+### Added
+
+- Nginx kubectl plugin has added a commend to `lint` your ingress resources. That way you can observe deprecated annotations or properties. To install the command go [here](https://github.com/kubernetes/ingress-nginx/blob/29f7d9a77ade24a7366ef4a6f258b8aeef50678c/docs/kubectl-plugin.md).
+
+### Changed
+
```

Comment:
the two changes should be below a general change that the version of the IC has been updated to 0.24.1, those two are details of the changelog of that, not changes we made

Reply:
also don't forget to link to upstream changelog/release notes for that

Reply:
But what do you think about mention it here? or just point to the upstream changelog?

Reply:
good but as sub points of the upstream change, like "notable changes"